src/app/containers/MediaPlayer/index.jsx

```diff
@@ -112,7 +115,30 @@ const MediaPlayerContainer = ({
     translations,
   );
 
+  const landscapeRatio = '56.25%'; // (9/16)*100 = 16:9
+  const portraitRatio = '177.78%'; // (16/9)*100 = 9:16
+  const StyledMessageContainer = styled.div`
+    padding-top: ${({ portrait }) =>
+      portrait ? portraitRatio : landscapeRatio};
```

Comment:
`portrait` is never true so this code is unused - think it's throwing the test coverage numbers too :)